src/global_logic.js

```diff
@@ -1853,6 +1853,14 @@ module.exports.addSkilldataToTotals = function (totals, comb, arml, buff) {
                     } else if (stype == 'rigaiBishojo') {
                         // skill is all allies not restricted to element
                         totals[key]["criticalDamageLimit"] += comb[i] * 0.05;
+                    } else if (stype == 'wand-count') {
+                        let numOfWand = 0;
+                        for (let i = 0; i < arml.length; i++) {
+                            if (comb[i] > 0) {
+                                numOfWand += arml[i].armType === "wand" ? comb[i] : 0;
+                            }
+                        }
+                        totals[key]["normalHP"] += amount * numOfWand;
```

Comment:
- [ ] Naming confuse point, epic weapon car-ong wand used stype: "epic" amount: "count-wand"
  - and outer scope has variable "countWand" it's unused if no car-ong wand set,
    we can set the variable before for-loop, then both skill can share the wand count info.
- [ ] reuse epic.countWandType function
  - ` totals[key]["normalHP"] += amount * epic.countWandType(arml, comb);` (not tested)
  epic module has generic function to generate another weapon types count.
  it assume future skill implemented count sword types etc.

Reply:
Changed on 42da935